dockernet/docker-compose-debug.yml

```diff
@@ -0,0 +1,173 @@
+version: "3"
```

Comment:
I don't think you need to duplicate every other service. I think you can change the debug compose file so that it only has stride one and then specify both files in the docker compose command

https://docs.docker.com/compose/production/#modify-your-compose-file-for-production